infra/auth0/inputs.tf

```diff
@@ -0,0 +1,8 @@
+# variable "FEIDE_CLIENT_ID" {
+#   type = string
+# }
+
+# variable "FEIDE_CLIENT_SECRET" {
+#   type      = string
+#   sensitive = true
+# }
```

Comment:
These should probably be pulled from Doppler in the future

Reply:
They are already in doppler for dev